src/Symfony/Component/Process/Process.php

```diff
@@ -429,6 +429,45 @@ public function wait(callable $callback = null)
         return $this->exitcode;
     }
 
+    /**
+     * Waits until the callback return true.
+     *
+     * The callback receives the type of output (out or err) and some bytes
+     * from the output in real-time while writing the standard input to the process.
+     * It allows to have feedback from the independent process during execution.
+     *
+     * @param callable $callback
+     *
+     * @throws RuntimeException When process timed out
+     * @throws LogicException   When process is not yet started
+     */
+    public function waitUntil(callable $callback)
+    {
+        $this->requireProcessIsStarted(__FUNCTION__);
+        $this->updateStatus(false);
+
+        if (!$this->processPipes->haveReadSupport()) {
+            $this->stop(0);
+            throw new \LogicException('Pass the callback to the Process::start method or enableOutput to use a callback with Process::waitUntil');
```

Comment:
Can you add a dot at the end of the exception message? Also, you need to quote PHP code:

```
Pass the callback to the "Process::start" method or "enableOutput" to use a callback with "Process::waitUntil".
```

Perhaps add `call` before enableOutput: `... method or call "enableOutput" ...`